src/lib/charts/BarChart/BarChart.tsx

```diff
@@ -4,7 +4,7 @@ import classNames from 'classnames';
 import { ChartJSData, ChartJSOptions } from '../chart.helpers';
 import { AxialChart, AxialChartProps } from '../AxialChart';
 
-import './roundedBarCharts';
+import './roundedBarCharts.ts';
```

Comment:
Is this change for tree shaking?

Reply:
oh no, haha that actually was a bad change. no extension is better so it can find the compile JS file instead.